sbi/utils/conditional_density.py

```diff
@@ -45,6 +46,9 @@ def eval_conditional_density(
         eps_margins2: We will evaluate the posterior along `dim2` at
             `limits[0]+eps_margins` until `limits[1]-eps_margins`. This avoids
             evaluations potentially exactly at the prior bounds.
+        return_raw_log_prob: If `True`, return the log-probability evaluated on the·
```

Comment:
It is not clear what "evaluated on the grid" really means. Maybe add a line in the general docstring above explaining why we are evaluating on a grid?